myAuth/views.py

```diff
@@ -0,0 +1,21 @@
+from .serializers import RegisterSerializer
+from rest_framework import mixins, viewsets, permissions, status
+from rest_framework.response import Response
+from django.contrib.auth.models import User
+
+
+class RegisterUser(mixins.CreateModelMixin, viewsets.GenericViewSet):
+    permission_classes = (permissions.AllowAny, )
+    serializer_class = RegisterSerializer
+
+    def create(self, request):
```

Comment:
Mixins provide default behaviour for CRUD operations, so you can exclude this method. However, password will be saved as plain text. You can "override" user creation directly in `RegisterSerializer` by creating method:

```
def create(self, validated_data):
        # call create_user on user object. Without this
        # the password will be stored in plain text.
        user = User.objects.create_user(**validated_data)
        return user
```

Reference this boilerplate: `https://github.com/Vivify-Ideas/python-django-drf-boilerplate`